ofrak_type/ofrak_type/error.py

```diff
@@ -2,6 +2,10 @@ class NotFoundError(RuntimeError):
     pass
 
 
+class MultipleResourcesFoundError(RuntimeError):
+    pass
+
+
```

Comment:
Let's move this error to `resource.py`, since it is specific to OFRAK resources: `Resource` is an `ofrak` thing.